docs/how-to/hip_runtime_api/multi_device.rst

```diff
@@ -0,0 +1,219 @@
+.. meta::
+    :description: This chapter describes how to use multiple devices on one host.
+    :keywords: ROCm, HIP, multi-device, multiple, GPUs, devices
+
+.. _multi-device:
+
+*******************************************************************************
+Multi-device management
+*******************************************************************************
+
+Device enumeration
+==================
+Device enumeration involves identifying all the available GPUs connected to the host system. A single host machine can have multiple GPUs, each with its own unique identifier. By listing these devices, you can decide which GPU to use for computation. This process helps in managing resources efficiently and optimizing performance. The host queries the system to count and list all connected GPUs, ensuring that the application can leverage the full computational power available.
+
+.. code-block:: cpp
+
+    #include <hip/hip_runtime.h>
+    #include <iostream>
+
+    int main()
+    {
+        int deviceCount;
+        hipGetDeviceCount(&deviceCount);
```

Comment:
Loop over the devices and print out the device properties, something like this:

https://github.com/ROCm/rocm-examples/blob/develop/HIP-Basic/device_query/main.cpp

Reply:
Maybe reuse the example code?

Reply:
I added something similar as in the example code.